### DISCUSSION TITLE
DEV SESSION 13 JAN --- 2000 UTC+8/1300 CET/1200 GMT/ 0700 EST

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#1935] [General]
Where: [via Matrix](https://matrix.to/#/#project-mesa:matrix.org)

Proposed Agenda:

- General updates 
- Google Summer of Code 2024 & NumFOCUS
- MEP Draft
- Front-end 

Suggested:
- Mesa-Geo: https://github.com/projectmesa/mesa-geo/issues/160
- https://github.com/projectmesa/mesa/discussions/1929 
- Release process discussion

Please post suggested agenda items or edit directly. 

/cc  @rht, @tpike3, @Corvince , @wang-boyu , @ankitk50, @EwoutH

**Comment by wang-boyu** [comment]
Would need to discuss this for Mesa-Geo: https://github.com/projectmesa/mesa-geo/issues/160

**Comment by EwoutH** [comment]
Thanks for the invite!

- I really want to discuss the examples. These are paramount in testing and showing new capabilities.
  - #1929
- I would like to talk further about the release process.
- @Corvince would you like to discuss the future of Mesa-interactive, and the position in/to Mesa?

**Reply by jackiekazil** [reply]
We discussed doing another working session around this. Are you open to something next week to focus just on this? (I think we need to discuss, and I don't want to take up the whole meeting.)

**Reply by EwoutH** [reply]
Yes, I’m quite flexible, could you suggest some dates?

**Comment by EwoutH** [comment]
I might sounds repetitive, but I also want to keep talking about a foundation / inter university collaboration / funding. My time at the TU Delft is likely (and hopefully, at least as a student) limited and at some point there the opportunity windows to advocate for this will be closing (with @quaquel involved there's some redundancy and seniority, but I don't know what role he's willing to take).

**Comment by EwoutH** [comment]
Sorry, some things came in between for me today.

For me personally, the Saturday early-afternoon time is just one of the worst times in the week for me to meet. It's breaking up the weekend, and Saturday it most of the time the day in the week that I try to not think about academic & work stuff, and it falls right in the middle of that day.

I understand the complexity of all the timezones involved, so I don't know if this situation can be resolved. Just wanted to give some context.

Maybe we could alternate the meetings between the Saturday, and some other moment, preferably a weekday.

**Comment by tpike3** [comment]
Meeting Notes: 

Google Summer of Code -- 
* NumFOCUS has assessed that doing a group submission is not optimal for the GSOC process and recommends each group submitting on their own. 
* [Applications open the 22 JAN for organizations to submit](https://summerofcode.withgoogle.com/)
* Concern is having coverage
      * @tpike3 can do August on
      * @jackiekazil  can do part of the June or July
      * @wang-boyu  can do part of June or July
 @EwoutH , @rht, @Corvince , @quaquel Would you be able to support mentorship of GSOC interns over the summer to ensure we have coverage. Please let me know by next week some time. 
 

* @tpike3 is taking the first pass at updating the Mesa-Geo tutorial for the American Association of Geographers conference. For this he will be working with Dr. Melissa LaDuke, a curriculum and instruction PhD, on developing new approaches for [adaptive andragogy](https://educationaltechnology.net/andragogy-theory-malcolm-knowles/). 

* On the front-end @ankit50 is going to finish #1902 which also needs updates for Mesa 2.2.2 and is building a gallery of examples based on @rlskoeser'a https://simrisk.pulcloud.io/riskybet Please comment at https://github.com/projectmesa/mesa/discussions/1980 

For the next dev session we will propose a different day of the week, probably same time.. let us know if you have a preferred day

**Reply by quaquel** [reply]
I am traveling in June and July. Not sure about August yet. Exactly what would be expected of mentorship?

**Reply by EwoutH** [reply]
I'm looking into a long offline vacation (possibly the [appalachian trail](https://en.wikipedia.org/wiki/Appalachian_Trail)) before my working life begins, so July and August will most likely be very limited. June would probably be possible to some extend.

All weekdays are fine for me!

**Reply by jackiekazil** [reply]
@quaquel - mentorship usually means regular meetings and providing guidance on direction as well as helping them get unblocked by helping the individual navigate the project -- eg finding the answers they need if they can't get them.

**Reply by jackiekazil** [reply]
@EwoutH good for you! It is good to do these things for ourselves.
